Platforms/QemuSbsaPkg/PlatformBuild.py

```diff
@@ -295,6 +295,8 @@ def PlatformPostBuild(self):
         args += " ARCH=" + self.env.GetValue("TARGET_ARCH").lower()
         args += " DEBUG=" + str(1 if self.env.GetValue("TARGET").lower() == 'debug' else 0)
         args += " SPM_MM=1 EL3_EXCEPTION_HANDLING=1"
+        args += " ENABLE_FEAT_HCX=1" # Features used by hypervisor
+        # args += " FEATURE_DETECTION=1" # Enforces support for features enabled.
         args += " BL32=" + os.path.join(op_fv, "BL32_AP_MM.fd")

```

Comment:
Also, what is this flag for?

Reply:
![image](https://user-images.githubusercontent.com/25697595/233730525-2b3e2d5f-cf5a-49b5-9358-f8f5e545d65b.png)

:).

That is the flag to tell TFA what BL32 image to use in the SPM_MM configuration.

Reply:
oh, man, i don't like this UI. I meant to ask for the "FEAURE_DETECTION" flag.

Reply:
it will basically panic BL31 if a request feature is not supported in hardware. Its nice for dev, but dont think it needs to eb on generally so i left it in commented out for posterity